### PR TITLE
fix(helm): add imagePullSecrets and fix imagePullPolicy

### LINKED ISSUE #14179: helm chart need imagePullSecret
Hi,

helm chart need imagePullSecret.

### PR DESCRIPTION
**Pull request by Hokwang**
SUMMARY
add imagePullSecrets and fix imagePullPolicy in helm chart

imagePullSecrets is needed for docker hub rate limit.
 
ADDITIONAL INFORMATION
 Has associated issue: fix #14179

### CODE REVIEW

**Comment by suddjian** [comment]
Hi, @Hokwang, thank you for the contribution.

Could you please provide some more information on what these changes are for, so that we can evaluate and get it merged? Preferably following the PR message template. Thank you.

**Comment by Hokwang** [comment]
@suddjian I edited.

File: helm/superset/values.yaml [hunk 1]
```diff
@@ -104,7 +104,10 @@ extraConfigMountPath: "/app/configs"
 image:
   repository: apache/superset
   tag: latest
```
Comment (danielewood) [inline helm/superset/values.yaml:106]: Given that you are setting the chart app version to 1.1.0, this should be set to the tag of 1.1.0
apache/superset:latest is master

See: https://hub.docker.com/r/apache/superset/tags

Comment (Hokwang) [reply helm/superset/values.yaml]: thanks, edited.

**Comment by Hokwang** [comment]
@danielewood can you review once again?

**Comment by danielewood** [comment]
> @danielewood can you review once again?

Unfortunately I'm just a contributor, not a project maintainer.

**Comment by Hokwang** [comment]
@suddjian can you take care of this?

**Review by craig-rueda — APPROVED** [review]

**Comment by Hokwang** [comment]
done by #14686

**Pull request closed without merging**